### ISSUE TITLE
[Accessibility] Support character-tracking in repl

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#32423]
Type: <b>Feature Request</b>

I understand that command history is not supported in an external repl outside the shell integration. It would be so nice if this could be supported in the future for screen reader users.

1. Execute `Python: Start REPL` in command palette.

1. In the Python repl, type `import os` and hit Enter.

1. Hit UpArrow.

1. Hit LeftArrow.

* Notes: Screen reader does not announce the command history.

VS Code version: Code - Insiders 1.84.0-insider (6a7a661757dec1983ff05ef908a2bbb75ce841e0, 2023-10-11T05:34:23.077Z)
OS version: Windows_NT x64 10.0.22621
Modes:
